maintenance_plan/models/maintenance.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import _, api, fields, models
+
+from odoo.exceptions import UserError
+
+
+class MaintenanceKind(models.Model):
+
+    _name = 'maintenance.kind'
+    _description = 'Maintenance Kind'
+
+    name = fields.Char(required=True, translate=True)
+    active = fields.Boolean('Active Kind', required=True, default=True)
+
+    _sql_constraints = [
+        ('name_uniq', 'unique (name)',
+         "Maintenance kind name already exists.")]
+
+
+class MaintenancePlan(models.Model):
+
+    _name = 'maintenance.plan'
+    _description = 'Maintenance Plan'
+
+    equipment_id = fields.Many2one(string='Equipment',
+                                   comodel_name='maintenance.equipment',
+                                   ondelete='cascade')
+    maintenance_kind_id = fields.Many2one(string='Maintenance kind',
+                                          comodel_name='maintenance.kind',
+                                          ondelete='restrict')
+
+    period = fields.Integer('Period', help='Days between each maintenance')
```

Comment:
This is what I meant:

```suggestion
    period = fields.Integer(help='Days between each maintenance')
```
Odoo will generate the string automatically.